glslc/src/file_compiler.cc

```diff
@@ -27,6 +30,32 @@
 namespace {
 using shaderc_util::string_piece;
 
+// A helper function to emit SPIR-V binary code as a list of hex numbers in
+// text form. Returns true if the compilation result is emitted successfully.
```

Comment:
"Returns true if a non-empty compilation result is emitted successfully."

It wasn't clear to me why *this function* should treat an empty result as a failure.  Just state that upfront.